app/src/org/commcare/android/adapters/AppManagerAdapter.java

```diff
@@ -0,0 +1,37 @@
+package org.commcare.android.adapters;
```

Comment:
Thanks! Added a class-level comment, do you feel like something more specific is needed?